pkg/users/account-create-dialog.js

```diff
@@ -209,9 +211,10 @@ export function account_create_dialog(accounts) {
 
         errors.user_name = validate_username(state.user_name, accounts);
 
-        return password_quality(state.password)
+        return password_quality(state.password, force)
                 .catch(ex => {
-                    errors.password = ex.message || ex.toString();
+                    errors.password = (ex.message || ex.toString()).replace("\n", " ");
+                    errors.password += "\n" + _("Press Create again to use the password anyway.");
```

Comment:
Use cockpit.format and `_("Press $0 again...")` with $0 == `_("Create")`. This:

 1. ensures that the button text is always consistent between the button itself and that error message, especially if one is translated and the other is not.
 2. The translation of this sentence is then shared with pkg/users/password-dialogs.js

Reply:
Good idea. Was thinking how to use one string for translation and this seems like a good approach. Updated.